ace-link-dashboard.el

```
@@ -89,5 +92,10 @@ When point is over an icon avy overlay could not be seen."
       (+ 2 point)
     point))
 
+(defun ace-link-dashboard--in-section-p ()
+  "Return t if point is in a known section of `dashboard-mode' buffer."
+  (ignore-error user-error
```

I don't consider ignoring errors a good practice.